anvil-rstudio-bioconductor/Dockerfile

```diff
@@ -20,8 +16,6 @@ RUN R -e "BiocManager::install(c('DataBiosphere/Ronaldo', 'shiny', 'bigrquery',
 
 ENV RSTUDIO_PORT 8001
 ENV RSTUDIO_HOME /etc/rstudio
-ENV TERRA_R_PLATFORM=$TERRA_R_PLATFORM
```

Comment:
Are `TERRA_R_PLATFORM` and `TERRA_R_PLATFORM_BINARY_VERSION` evs not needed anymore?

Reply:
don't believe so, @nturaga can confirm

Reply:
No need for this anymore because we inherit everything from bioconductor/bioconductor_docker images. These images have env variables we can use to find version identifiers and platform and other binaries related information.